src/reducers/projects.js

```diff
@@ -25,13 +25,14 @@ const newProject = Immutable.fromJS({
     javascript: '',
   },
   enabledLibraries: new Immutable.Set(),
+  minimizedComponents: new Immutable.Set(),
```

Comment:
Does the name `minimizedComponents` still make sense in the context of a project? Honestly I don’t have a better suggestion, just throwing the question out there…